marimo/_cli/export/commands.py

```diff
@@ -57,6 +57,16 @@ def watch_and_export(
 
     def write_data(data: str) -> None:
         if output:
+            output_path = Path(output)
+            # Check if the file exists
+            if output_path.exists():
```

Comment:
we should probably only check once when `watch` is enabled, or not check at all.

could we also check if `GLOBAL_SETTINGS.YES`, is set and skip?

maybe we could pull this out to a util:

```python
def maybe_prompt_to_overwrite(path: Path) -> bool:
    if GLOBAL_SETTINGS.YES:
        return True

    # Check if not in an interactive terminal
    # default to False
    if not sys.stdin.isatty():
        return True
    
    if output_path.exists():
         return True

    confirmed = click.confirm(
        f"Warning: The file '{output}' already exists. Overwrite?",
        default=False,
    )
    
    if not confirmed:
        click.exit()
```